### PR TITLE
added printing out of a confusion matrix

### PR DESCRIPTION
**Pull request by calz1**
Shows a confusion matrix of what was predicted versus what the truth (actuals) were. Useful when you want to know false positives vs false negatives and with imbalanced data

### CODE REVIEW

File: auto_ml/utils_scoring.py [hunk 1]
```diff
@@ -35,6 +37,14 @@ def advanced_scoring_classifiers(probas, actuals, name=None):
             predicted_labels.append(0)
     print(format(accuracy_score(y_true=actuals, y_pred=predicted_labels) * 100, '.1f') + '%')
 
+    
+    print('\nHere is a confusion matrix showing predictions and actuals by label')
+    #it would make sense to use sklearn's confusion_matrix here but it apparently has no labels
```
Comment (ClimbsRocks) [inline auto_ml/utils_scoring.py]: Thanks much for explaining the design decision, and linking to a useful source for anyone that wants to do any future work related to this.

**Comment by ClimbsRocks** [comment]
This is great! Thanks so much for the PR. This has been on my mental backlog of things to add, but I had no idea it would be so easy. 

Have you looked at [scikit-plot](https://github.com/reiinakano/scikit-plot)? They had a couple of useful looking charts I hope to get added to auto_ml at some point. Just curious if you had any experience with them. 

Thanks for the PR! I love the comments in your code- they're quite helpful.

**Comment by ClimbsRocks** [comment]
I'll upload to pypi in a minute- version 2.0.1

File: auto_ml/utils_scoring.py (hunk 1 above)
Comment (calz1) [inline auto_ml/utils_scoring.py]: I figured I should explain why I wasn't using the built-in function. Conveniently, the guy on StackExchange basically solved it.

**Comment by calz1** [comment]
Cool! I am not familiar with scikit-plot but at first glance it has some relevant stuff. Design-wise, how were you thinking plots would be handled? If you were running all this in a Jupyter notebook it would be fine to display to console, but maybe they should go to file system by default. I guess it would be similar to how you save a .CSV to the file system with the grid search results.

**Comment by ClimbsRocks** [comment]
Ooh, I like that idea. That should ensure compatibility across different environments (both gui and non-gui). We could also nest it behind some if checks, and allow the user to pass in a param to `.train` that says whether they want charts printed or not. 

But your idea of writing to file (with some logging to let people know about that feature and where to find the file), sounds like an easy solution that takes care of most of the edge cases I can think of right now.

**Comment by calz1** [comment]
Is ClimbsBytes@gmail.com still a good email address to contact you at? I have some implementation questions...

**Pull request merged**